src/nemspecs.f

```diff
@@ -120,7 +120,7 @@ RECURSIVE SUBROUTINE NEMSPECS
      .          ( TAGN(1:8) .EQ. TAGNRV(JJ) ) .AND.
      .          ( NODE .GE. ISNRV(JJ) ) .AND.
      .          ( NODE .LE. IENRV(JJ) ) ) THEN
-              NREF = NRV(JJ)
+              NREF = INT(NRV(JJ))
```

Comment:
Sometimes int sometimes nint? 

When not specified, int is used, right? So are you sure all the nint() calls are correct?

Reply:
NINT is for converting from a real to an integer.  INT is used when converting from an integer*8 to an integer*4.